lib/apiconfig/apiconfig.go

```diff
@@ -76,3 +81,17 @@ func NewCalicoAPIConfig() *CalicoAPIConfig {
 		},
 	}
 }
+
+// IsAlphaFeatureSet checks if the comma separated features have the
+// name set in it.
+func IsAlphaFeatureSet(features, name string) bool {
```

Comment:
Just trying to follow the flow of the config here.  Shouldn't this really be a method attached to the apiconfig struct - this ties in with the question that you are accessing the ALPHA_FEATURES environment somewhere in the code whereas everywhere else you access it via the apconfig.

Also, should this return false for the Service account feature if the datastore type is not KDD?

Reply:
yes on this too. I do plan to make this a method attached to apiconfig spec. that will make it easier to take into account the datastore.